### DISCUSSION TITLE
Getting Error with missing Folder from CMakeLists.txt

### DISCUSSION THREAD

**Discussion by Helder1981** [darthcloud/BlueRetro#820] [Q&A]
I am trying to compile from source code using MSYS64 on windows and when I run the commands to configure the files to build there is a missing folder required to finish the files and I can not find it anywhere in the included folders.

**_CMake Error at CMakeLists.txt:5 (include):
  include could not find requested file:_**

   **_ /tools/cmake/project.cmake_**

Does anyone know where to get these files?

**Comment by darthcloud** [comment]
Did you follow instructions? 
https://github.com/darthcloud/BlueRetroRoot

on windows you will need use PowerShell and use install.ps1 script instead of the .sh one.

**Comment by Helder1981** [comment]
You mean the ones for Ubuntu? No I did not but since you pointed that link out I discovered that the the files or instructions must be in there and it led me to this here:
https://github.com/espressif/esp-idf/tree/8fc8f3f47997aadba21facabc66004c1d22de181

Which I assume is what I need to do to install this on windows to get the missing files to compile this. I don't want to come off as an ass but these things are not a given for people who are not familiar with programming or what requirements are needed to even compile anything. 

This is not just on your github project but every other project I stumble upon. It would be nice to have some simple instructions on what to install and what commands are required to get it to compile and have it one of the entries like the build guide pages.

Thanks for pointing me in the right direction though I'm sure I can figure the rest out.

**Reply by darthcloud** [reply]
I don't want to come off as an ass but the page I gave you (and that is linked in the wiki) have literally a step by step guide on how to get the environment setup and then how to build an image.

Their is a reason why nobody use Windows for development. Cross compiling for an embedded device on Windows is a PITA.

So don't go around and expect to have instruction for that ever.

People making free project really don't own you anything.
If you use a different OS than the author that's for you to figure out.

**Comment by darthcloud** [comment]
@Helder1981 I made some instruction for Windows
https://github.com/darthcloud/BlueRetroRoot/blob/master/README.windows

I still got to update the main README.md to link to it. But I'll do that once I got feedback from a few people.
